### PR TITLE
Updated version to `5.1.0`, default `SIBLING_CONTAINERS_ENABLED` to true

### PR DESCRIPTION
**Pull request by mserranom**
## Description

- Updates `config-seed/version` to `5.1.0`.
- `SIBLING_CONTAINERS_ENABLED` now defaults to `true`.
- Updated changelog, including and amend on the date of the previous entry.

## Contributor Agreement

- [x] I confirm I have signed the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 2024-07-27
+## 2024-07-17
+### Added
+- Updated default [`version`](https://github.com/overleaf/toolkit/blob/master/lib/config-seed/version) to `5.1.0`.
+- `SIBLING_CONTAINERS_ENABLED` is now set to `true` for new installs in [`config-seed/overleaf.rc`](https://github.com/overleaf/toolkit/blob/master/lib/config-seed/overleaf.rc).
+
```
Comment (das7pad) [inline CHANGELOG.md:7]:
```suggestion
  We strongly recommend enabling the [Sandboxed Compiles feature](https://github.com/overleaf/toolkit/blob/master/doc/sandboxed-compiles.md) for existing installations as well.

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+## 2024-07-17
 ### Added
```
Comment (das7pad) [inline CHANGELOG.md:9]: WDYT about merging these entries?

**Review by das7pad — APPROVED** [review]
:rocket:

File: CHANGELOG.md (hunk 2 above)
Comment (mserranom) [reply CHANGELOG.md:9]: yeah, good idea.

**Review by das7pad — APPROVED** [review]

**Pull request merged**